src/components/App/App.js

```diff
@@ -32,6 +33,16 @@ class App extends Component {
           <NavBar />
         </header>
         <Switch>
+          <Route path='/birds/:id' render={({ match }) => {
+            const bird = this.state.regionBirds.find(bird => bird.speciesCode === match.params.id)
+
+            if(!bird) {
+              return (<div>That bird doesn't exist!</div>)
```

Comment:
Would it make more sense for this to be a h2 or p, rather than a div?

Reply:
Sure! I'll change to an H2